frontend/src/features/menus/ShareMenu.tsx

```diff
@@ -40,6 +42,13 @@ const ShareMenu = ({
     onClose(event);
   };
 
+  const shareYoutubeLink = (event: React.MouseEvent<HTMLElement>) => {
+    if (youtubeLink && navigator.share) {
+      navigator.share({ url: youtubeLink });
+    }
```

Comment:
`navigator.share` is not available on all browsers (e.g on Firefox desktop, or on Chromium Linux). So we should use the clipboard as a fallback, or hide the menu item completely in that case.